frontend/src/libs/components/card/styles.module.scss

```diff
@@ -0,0 +1,43 @@
+.item {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 317px;
+  height: 83px;
```

Comment:
you can use min/max width/height values, or achieve this by the padding (depends on the layout), but don't use fixed width and height for such cases 
```suggestion
```

Reply:
done